app/templates/index.html

```diff
@@ -41,24 +41,6 @@
                   <input type="text" name="service_id" id="service_id" class="text-box">
                   <input type="submit" value="Search" class="button-save">
               </form>
-
-              <form action="{{ url_for('.find_supplier_services') }}" method="get" class="question">
-                  <label class="question-heading" for="supplier_id_for_services">Find services by supplier ID</label>
-                  <p class='hint'>
-                    eg 54321
-                  </p>
-                  <input type="text" name="supplier_id" id="supplier_id_for_services" class="text-box">
-                  <input type="submit" value="Search" class="button-save">
-              </form>
-
-              <form action="{{ url_for('.find_supplier_users') }}" method="get" class="question">
-                  <label class="question-heading" for="supplier_id_for_users">Find users by supplier ID</label>
-                  <p class='hint'>
-                    eg 54321
-                  </p>
-                  <input type="text" name="supplier_id" id="supplier_id_for_users" class="text-box">
-                  <input type="submit" value="Search" class="button-save">
-              </form>
```

Comment:
We don't need to delete the view stuff because it's still used elsewhere, right?  (Just making sure.)